docs/sources/enterprise/access-control/permissions.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Permissions"
+description = "Understand permissions in fine-grained access control"
```

Comment:
```suggestion
description = "Understand fine-grained access control permissions"
```
As odd as it is, the weight (importance) falls on the final word in this phrase.